src/main/java/core/basesyntax/impl/CreateReportServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.impl;

```

Comment:
correctly to put impl package inside service package couse this is implementation of service